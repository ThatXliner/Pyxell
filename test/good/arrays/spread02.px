
print [...[1, 3], 2]
print [true..false step -1, ...[]]
print [...[] step 2, ...[4/3]]
