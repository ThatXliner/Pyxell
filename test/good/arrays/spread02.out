[1, 3, 2]
[true, false]
[4/3]
